styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,64 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: calc(((100vw - 40px) / 3) * 2);
+
+  @include breakpoint($bp-med) {
+    min-height: calc(((100vw -  60px) / 2) / 3 * 2);
+    min-width: 50%;
+    flex-basis: 50%;
+  }
+
+  @include breakpoint($bp-large) {
+    min-height: 400px;
+  }
+}
+
+.testimonial_text {
+  padding: 30px;
```

Comment:
@rpama I think this is an improvement, but now I'm confused because the image gets all stretchy and I'm not sure if this is desirable. Thoughts? 

![screen shot 2017-06-23 at 3 27 59 pm](https://user-images.githubusercontent.com/12160398/27499248-9361f1aa-5828-11e7-8e51-8623ebea70fe.png)